gov2/sdk/README.md

```diff
@@ -0,0 +1,36 @@
+# AWS SDK for Go V2 code examples for the SDK 
+
+## Purpose
+
+These examples show ways to work with Go-specific aspects of the
+AWS SDK for Go V2. 
+
+### `config.go`: Configuring the credentials used by the SDK
+
+This example demonstrates
+
+* Loading the default configuration
+* Assuming a role using STS 
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)
+in the AWS SDK for Go Developer Guide.
+
+### Notes
+
+- We recommend that you grant this code least privilege,
+  or at most the minimum permissions required to perform the task.
+  For more information, see
+  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the AWS Identity and Access Management User Guide.
+- This code has not been tested in all AWS Regions.
+  Some AWS services are available only in specific
+  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+- Running this code might result in charges to your AWS account.
+
```

Comment:
Although you haven't got unit tests yet, consider adding a section about the unit tests. If you're planning to explain how to run unit tests at the service level, you could just say that here.

Include a section ##Contributing - To propose a new code example to the AWS documentation team, see the CONTRIBUTING.md (https://github.com/awsdocs/aws-doc-sdk-examples/blob/master/CONTRIBUTING.md). The team prefers to create code examples that show broad scenarios rather than individual API calls.